hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMasterCommandLine.java

```diff
@@ -139,7 +148,15 @@ public int run(String args[]) throws Exception {
     if ("start".equals(command)) {
       return startMaster();
     } else if ("stop".equals(command)) {
-      return stopMaster();
+      if (shutDownCluster) {
+        return stopMaster();
+      }
+      System.err.println(
+        "To shutdown the master run " +
+        "hbase-daemon.sh stop HMaster or send a kill signal to " +
+        "the HMaster pid, " +
+        "and to stop HBase Cluster run stop-hbase.sh or add --forceStop");
```

Comment:
nit: `and to stop HBase Cluster either run stop-hbase.sh or add --forceStop with current command`